electron/main.ts

```diff
@@ -552,48 +553,73 @@ interface Tools {
   wine: string
 }
 
+const runWineTricks = async (
+  prefix: string,
+  wine: string,
+  isProton: boolean
+) => {
+  const winetricks = `${heroicToolsPath}/winetricks`
+  const newProtonWinePath = wine.replace(
+    new RegExp('proton' + '$'),
+    'files/bin/wine64'
+  )
+  const oldProtonWinePath = wine.replace(
+    new RegExp('proton' + '$'),
+    'dist/bin/wine64'
+  )
+
+  const protonWinePath = existsSync(newProtonWinePath.replaceAll("'", ''))
+    ? newProtonWinePath
+    : oldProtonWinePath
+
+  const wineBin = isProton ? protonWinePath : wine
+
+  let winePrefix = prefix.replace('~', userHome)
```

Comment:
Having a non-absolute prefix path causes problems elsewhere (`launch` doesn't do any checks for it, for example).

I'd say instead of having this `.replace` everywhere, we should make sure it's absolute when saving the settings and replace things like ~ there. But that can of course be a different PR

Reply:
yes, we need to look into that. Maybe if we expose the userHome on frontend we could get the right folder.

Reply:
Done